### PR TITLE
feat(syntax): ledger

### PR DESCRIPTION
**Pull request by matoous**
Add support for ledger language. Ledger is tool for command line accounting,
see: https://www.ledger-cli.org/

### CODE REVIEW

File: languages.toml [hunk 1]
```diff
@@ -499,3 +499,13 @@ file-types = ["git-rebase-todo"]
...
+
+[[language]]
+name = "ledger"
+scope = "source.ledger"
+injection-regex = "ledger"
+file-types = ["ldg"]
+roots = [ ]
+comment-token = ";"
+
+indent = { tab-width = 2, unit = "  " }
```
Comment (kirawi) [inline languages.toml:511]: Needs a newline.

Comment (matoous) [reply languages.toml]: Done, thanks! Would you also please know how to commit the submodule? 🤔 My git skills are lacking.

**Comment by kirawi** [comment]
https://docs.helix-editor.com/guides/adding_languages.html

**Comment by matoous** [comment]
@kirawi don't mind me, I am stupid and completely missed that ledger is already supported 🤦 closing.

**Pull request closed without merging**